histogram.c

```diff
@@ -0,0 +1,92 @@
+//---------------------------------------------------------------------------------------------
+//
+// Copyright (c) 2018, fmad engineering llc 
+//
+// The Creative Commons BY-NC 4.0 International License see LICENSE file for details 
+// 
+// Histogram generation from pcap
+//
+//---------------------------------------------------------------------------------------------
+
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+#include <malloc.h>
+#include <errno.h>
+#include <sys/stat.h>
+#include <sys/mman.h>
+#include <sys/shm.h>
+#include <fcntl.h>
+#include <locale.h>
+#include <linux/sched.h>
+#include <pthread.h>
+
+#include "fTypes.h"
+#include "fProfile.h"
+#include "histogram.h"
+
+
+PacketInfoBulk_t* PktInfoBulk_Alloc(u32 MaxPkts)
```

Comment:
all the public functions in this file need a consistant prefix.

PkInfoBulk_Alloc -> PktInfo_BulkAlloc